guides/v2.1/cloud/project/ece-tools-upgrade-project.md

```diff
@@ -0,0 +1,92 @@
+---
+group: cloud
+title: Upgrade to ece-tools
+version: 2.1
+github_link: cloud/project/ece-tools-upgrade-project.md
+functional_areas:
+  - Cloud
+  - Upgrade
+---
+If you still use a version of Cloud that does not contain the`ece-tools`package, then your project requires an _upgrade_. We deprecated the `magento/magento-cloud-configuration` and `magento/ece-patches` packages in favor of the `ece-tools` package.
+ 
+ You must perform a one-time, manual step to update the `magento/magento-cloud-metapackage` version constraint in the `composer.json` file, located in the root directory. This constraint enables updates for Magento Cloud metapackages—including removing deprecated packages—without upgrading your current {{site.data.var.ee}} version.
+ 
+ If you have already performed this step and you need to update your current `ece-tools` package, see [Update ece-tools]({{page.baseurl}}/cloud/project/ece-tools-update.html).
+
+{% include cloud/note-upgrade.md %}
+
+### Remove deprecated packages
+Before performing an upgrade to use the`ece-tools`package, check the `composer.json` file for the following deprecated packages and remove them:
+
+-  `magento-cloud-configuration`
+-  `magento/ece-patches`
+ 
+### Update the metapackage {#metapackage}
+Each {{site.data.var.ee}} version requires a different constraint based on the following:
+
+```
+>=current_version <next_version
+```
+
+You can always find the latest metapackage constraint in the [`magento-cloud` template](https://github.com/magento/magento-cloud/blob/master/composer.json).
+
+The following example places a constraint for the Magento Cloud metapackage to any version greater than or equal to the current version 2.2.0 and lower than next version 2.2.1:
+
+```json
+"require": {
+    "magento/magento-cloud-metapackage": ">=2.2.0 <2.2.1"
+},
+```
+
+### Upgrade the project
+
+To upgrade your project to use the `ece-tools` package, you need to update the metapackage, update the `.magento.app.yaml` hooks properties, and run a composer update.
```

Comment:
I thought we avoided colloquialisms like "run". Also, "composer" should be capitalized.